### DISCUSSION TITLE
Using iOS app on Mac

### DISCUSSION THREAD

**Discussion by WinterLoki** [LeoKlaus/plappa#48] [Feature Requests]
I see that the Mac App is on the roadmap. Would it be possible/any easier to have the iOS app be available on Apple Silicon Macs in the mean time?

**Comment by LeoKlaus** [comment]
I've tried it but am really not happy with the result. 
While I understand that you might not mind having to awkwardly deal with touch gestures on a mac, it's really not a state I want to ship a product in.

Porting plappa to mac shouldn't be that big of a task though, most of the views can be reused, it's mainly the "now playing" screen and bar that have to be remade. I can't give you any timeline or expected release date, but I can tell you that I've already started working on this. For now, there are still some bugs and important features I want to implement before continuing work on the mac app, but it's still pretty high up my todo list.

**Comment by LeoKlaus** [comment]
Closing this as it's not something I plan on doing for now. The mac app is still in active development.
